docs/integrations/scheduled-commands.md

```diff
@@ -0,0 +1,60 @@
+2. ***Rate Limiting*** - The command cannot perform the required action because the instance has reached its rate limit. Scheduled commands enable to set the command to try again later to check if the rate limit was removed, so the command can be executed.
+3. ***Concurrency*** - The command cannot perform the required action because the instance has reached its concurrence limit. Scheduled commands enable to set the command to try again later (preferably with an exponential backoff).
+
+### YAML Prerequisite
+* ***Integration***: In the integration yml, under the command root add `polling: true`.
+* ***Script***: In the script yml, in the root of the file add `polling: true`.
+
+See for an example: Autofocus V2 `autofocus-samples-search` command.
+
+### ScheduleMetadata
+The `ScheduleMetadata` is an optional class that enables scheduling commands via the command results.
+
+#### ScheduleMetadata Class Arguments
+| Arg               | Type   | Description                                                                                                                                                                                |
+|-------------------|--------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| command                | str    | The command that'll run after next_run_in_seconds has passed.
+| next_run_in_seconds    | int    | How long to wait before executing the command.
+| args                   | dict   | Arguments to use when executing the command.
+| timeout_in_seconds     | int    | Number of seconds until the polling sequence will timeout.
+
+When provided to a [CommandResults](./code-conventions#commandresults) it transforms its result to a ***schedule result***.
+When the time comes for the next command to run, it'll be executed.
+The scheduled command can return another ***schedule result***, that will schedule another schedule command and so on.
+
+The schedule sequence will be complete when either one of 3 terminating actions happen:
+
+1. ***Done*** - The schedule sequence is done, indicated by a command execution without any schedule result.
+2. ***Error*** - The command encountered an error, indicated by an error result.
+3. ***Timeout (automatically handled)*** - The schedule sequence reached the timeout, in which case a timeout error entry will be returned automatically.
+
+#### Code Example
+In the example below if the `status` is not `complete` then a result with `schedule_config` will be returned to trigger in 60 seconds a poll for the search. This will be done in the next run as well, and again until its status is complete.
```

Comment:
```suggestion
In the example below, if the `status` is not `complete` then a result with `schedule_config` will be returned which will trigger in 60 seconds a poll for the search. This will be done in the next run as well, and again until its status is complete.
```